docker/Dockerfile

```diff
@@ -25,23 +27,29 @@ RUN chown -R espressopp:espressopp /home/espressopp/espressopp /home/espressopp/
 USER espressopp
 
 WORKDIR /home/espressopp/espressopp
-RUN mkdir build
+RUN maintainer/check_codestyle.sh ${CODESTYLE} ${TRAVIS_BRANCH} ${TRAVIS_PULL_REQUEST} ${TRAVIS_COMMIT} ${TRAVIS_COMMIT_RANGE}
 
-WORKDIR build
-RUN ccache -z
-RUN cmake -DCMAKE_INSTALL_PREFIX=/usr -DEXTERNAL_BOOST=$EXTERNAL -DEXTERNAL_MPI4PY=$EXTERNAL -DWITH_XTC=ON ${COVERAGE:+-DUSE_GCOV=ON} ..
-RUN make VERBOSE=1 -j2 all ${BUILD_UG:+ug doc ug-pdf}
-RUN ccache -s
-RUN make test
-RUN make install DESTDIR=${PWD}
-RUN cd .. && if [ ${COVERAGE} ]; then \
-  if [ ${CC} = clang ]; then \
-    $HOME/.local/bin/codecov --gcov-exec "llvm-cov gcov"; \
+RUN if [ "${CODESTYLE}" = ON ]; then \
+      exit 0; \
   else \
-    $HOME/.local/bin/codecov; \
-  fi; \
-fi && cd -
+      mkdir build && \
+      cd build && \
+      ccache -z && \
+      cmake -DCMAKE_INSTALL_PREFIX=/usr -DEXTERNAL_BOOST=$EXTERNAL -DEXTERNAL_MPI4PY=$EXTERNAL -DWITH_XTC=ON ${COVERAGE:+-DUSE_GCOV=ON} .. && \
+      make VERBOSE=1 -j2 all ${BUILD_UG:+ug doc ug-pdf} && \
+      ccache -s && \
+      make test && \
+      make install DESTDIR=${PWD} && \
+      cd .. && \
+      if [ ${COVERAGE} ]; then \
+        if [ ${CC} = clang ]; then \
+          $HOME/.local/bin/codecov --gcov-exec "llvm-cov gcov"; \
+        else \
+          $HOME/.local/bin/codecov; \
+        fi; \
+      fi; \
+  fi && cd -
 USER root
-RUN make install
+RUN [ "${CODESTYLE}" = ON ] && exit 0 || make install
```

Comment:
Why we run again `make install`, this time as a root?

Reply:
The first install tests if `DESTDIR` is honored!

Reply:
```suggestion
RUN [ "${CODESTYLE}" = ON ] || make install
```